src/CreateHandler/CreateHandler.php

```diff
@@ -156,6 +159,7 @@ private function discoverNamespaceAndPath(string $class, array $autoloaders) : a
     }
 
     /**
+     * @param string $class
```

Comment:
Basically revert all changes in that file and everywhere where you changed PHPDocs, please.